packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.vue

```diff
@@ -0,0 +1,84 @@
+<script lang="ts" setup>
+import { useSlots } from "vue";
+
+const props = withDefaults(
+  defineProps<{
```

Comment:
For flexibility we should export the Props as standalone types so they can be imported by projects.
Same for the OnyxPageLayout